src/robusta/integrations/receiver.py

```diff
@@ -36,6 +37,12 @@
 )
 
 
+class ValidationResponse(BaseModel):
+    http_code: int = 200
+    error_code: Optional[int] = None
```

Comment:
error code-> error message looks like a python enum

Reply:
you're right, changing to enum